modules/aleph/src/Aleph/Entity/AlephRequestResponse.php

```diff
@@ -94,6 +108,7 @@ public function success() {
   public static function createRequestResponseFromXML(\SimpleXMLElement $xml) {
     $response = new self();
 
+    $response->setXML($xml);
```

Comment:
Why do we need to set the XML here?

1. The property never seems to be accessed
2. Accessors are public but never called
3. The property is type hinted to be a string but the value set is a `\SimpleXMLElement`

I cannot tell how it relates to the issue at hand. I vote to remove all changes to this file.

Reply:
I agree. Leftover. :)